assets/api-swagger/swagger.json

```diff
@@ -777,38 +777,142 @@
         "tags": [
           "Profiles"
         ],
-        "summary": "Merge profiles with duplicate email and/or id",
-        "description": "Merge the profile on email and / or id",
+        "summary": "Merge profiles based on the provided identifiers.",
+        "description": "Merge customers_to_merge profiles into the survivor profile using the allowed identifiers (customer_id, profile_hash, email_address and phone_number).",
```

Comment:
What is the 'survivor' in this context? Maybe a term users will be familiar with that I haven't heard before

Reply:
Yes, we introduced two new terms: `survivor` and `customers_to_merge`. When we merge profiles, only one profile will remain, that is the `survivor`. All `customers_to_merge` profiles will be merged into the `survivor`.